src/odemis/gui/model/__init__.py

```diff
@@ -1051,25 +1051,24 @@ class FastEMAcquisitionGUIData(MicroscopyGUIData):
     # Parameters of the scintillators according to the technical drawing of the sample carrier
     # (positions are given relative to top left of sample carrier)
     # Scintillator arrangement on the sample carrier (9 is top left, 1 is bottom right):
-    # 9 8 7
-    # 6 5 4
-    # 3 2 1
+    # 3 6 9
```

Comment:
These lines seem to contradict. 9 is top right now isn't it? and 1 is bottom left. Or is this a matter of perspective? (like looking from behind my screen, than maybe that should also be commented.)

Reply:
Indeed, I think the comment just above should be updated too.

Reply:
Indeed, that is confusing. I deleted it.